### PR TITLE
Wazo 1101 get refresh tokens

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

**Comment by pc-m** [comment]
Depends-On: https://github.com/wazo-platform/wazo-auth-client/pull/4

File: integration_tests/suite/helpers/fixtures/http.py [hunk 1]
```diff
@@ -79,10 +79,14 @@ def token(**token_args):
     def decorator(decorated):
         @wraps(decorated)
         def wrapper(self, *args, **kwargs):
+            token_args.setdefault('backend', 'wazo_user')
+            token_args.setdefault('expiration', 1)
+            token_args.setdefault('client_id', _random_string(20))
```
Comment (fblackburn1) [inline integration_tests/suite/helpers/fixtures/http.py]:
```suggestion
            if token_args.get('access_type'):
                token_args.setdefault('client_id', _random_string(20))
                token['client_id'] = token_args['client_id']
```

File: integration_tests/suite/helpers/fixtures/http.py [hunk 2]
```diff
@@ -79,10 +79,14 @@ def token(**token_args):
...
             username = token_args.pop('username')
             password = token_args.pop('password')
             client = self.new_auth_client(username, password)
             token = client.token.new(**token_args)
+            token['client_id'] = token_args['client_id']
```
Comment (fblackburn1) [inline integration_tests/suite/helpers/fixtures/http.py]:
```suggestion
```

**Comment by fblackburn1** [comment]
recheck

**Pull request merged**